### PR TITLE
closes 166 minor styling

### PR DESCRIPTION
**Pull request by Tremirre**
Fulfilled the tasks outlined in #166.

To incorporate process title I had to slightly adjust the layout, now it looks like this:
image

_Statistics_ stage plots have been changed to bar plots:
image

Decimals & thousand delimiters have been set for all numeric displays, excluding plotly's plots which have their own custom formattings.

Added info icons with descriptions whenever I found necessary.

Let me know if something should be changed or some new tooltips added

### CODE REVIEW

**Comment by Tremirre** [comment]
also: I moved the function for making tooltips to `dashboard.pages.components` module for consistency

**Comment by dabrze** [comment]
Double check with our expert if he wants bar chart in all three plots. I like the previous plots better...

**Review by azywot — COMMENTED** [review]
Great job! Looks much cleaner and friendly for HTS Insight newcomers 😎

Spotted three more places where info icon might be a good idea:
-  Download xlsx file with images: `Note: this may take a minute since the file contains a concentration plot for every compound.`
- Screening projection input: `Minimum 3 experiments are required for projections generation.`, we can also add a blocker
- New SMILES input file (since this one is not any output from our processes): `CSV file containing columns: "EOS", "smiles".`

File: dashboard/visualization/plots.py [hunk 1]
```diff
@@ -330,14 +329,12 @@ def plot_row_col_means(plate_array: np.ndarray) -> go.Figure:
...
         fig.add_trace(
-            go.Scatter(
+            go.Bar(
                 x=ticks,
                 y=means,
                 error_y=dict(
                     type="data", array=stds, color="gray", thickness=0.5, width=2
                 ),
-                mode="markers",
-                marker_color="blue",
```
Comment (zuzg) [inline dashboard/visualization/plots.py:340]: I think red color is quite misleading here, as it is the same as `control neg`

**Review by zuzg — COMMENTED** [review]
Nice job, added small suggestion + definitely I would ask KŻ if these barplots are necessary (especially the one with control values)

File: dashboard/visualization/plots.py (hunk 1 above)
Comment (Tremirre) [reply dashboard/visualization/plots.py:340]: I changed the coloring to more neutral one (that still allows to differentiate between the two)
image

**Comment by Tremirre** [comment]
Added the following error message when user uploads less than three files in **Data Projection** first stage
image

Extended different descriptions as per @azywot recommendation

**Review by zuzg — APPROVED** [review]

**Review by AndrzejKaj — APPROVED** [review]

**Review by azywot — APPROVED** [review]

**Pull request merged**